$NetBSD: patch-Makefile,v 1.2 2012/06/03 12:38:39 othyro Exp $

Add path to boost headers.
Use pkgsrc infrastructure for libdl.

--- Makefile.orig	2012-05-08 20:00:07.000000000 +0000
+++ Makefile
@@ -13,7 +13,7 @@ target  := ethos
 # compiler
 c       := $(compiler) -std=gnu99
 cpp     := $(subst cc,++,$(compiler)) -std=gnu++0x
-flags   := -I. -O3 -fomit-frame-pointer
+flags   := -I. -O3 -fomit-frame-pointer -I${PREFIX}/include/boost
 link    := -s
 objects := libco
 
@@ -31,7 +31,7 @@ endif
 # platform
 ifeq ($(platform),x)
   flags += -march=native
-  link += -ldl -lX11 -lXext
+  link += ${DL_LIBS} -lX11 -lXext
 else ifeq ($(platform),osx)
 else ifeq ($(platform),win)
   link += $(if $(findstring console,$(options)),-mconsole,-mwindows)
